### PR TITLE
feedback-bugs: Comment line break bug fixed.

### PR DESCRIPTION
**Pull request by tombassrosenfeld**
Comment component created and implemented.

### CODE REVIEW

File: src/components/Feedback.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+ import React from 'react';
```
Comment (oliward) [inline src/components/Feedback.js]: Here.

File: src/components/instructor/StudentMarkingForm.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ class StudentMarkingForm extends Component {
     }
 
     render() {
+
```
Comment (oliward) [inline src/components/instructor/StudentMarkingForm.js]: Here.

File: src/components/instructor/StudentMarkingTask.js [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import StudentMarkingForm from "../../containers/instructor/StudentMarkingForm";
+import Comments from "../Comments";
 const Parser = require('html-react-parser');
 
+
```
Comment (oliward) [inline src/components/instructor/StudentMarkingTask.js]: Here.

File: src/components/instructor/StudentMarkingTask.js [hunk 4]
```diff
@@ -1,8 +1,12 @@
...
 export default ({question, answer, taskID, topicID, student }) => (
+
```
Comment (oliward) [inline src/components/instructor/StudentMarkingTask.js]: Here.

File: src/components/instructor/StudentMarkingTask.js [hunk 5]
```diff
@@ -18,12 +22,12 @@ export default ({question, answer, taskID, topicID, student }) => (
...
+		<Comments
+			title="Previous Comments"
+			commentArray={answer.get('feedback')}
+		/>
+
 	</div>
-)
\ No newline at end of file
+)
+
```
Comment (oliward) [inline src/components/instructor/StudentMarkingTask.js]: Here.

**Comment by tombassrosenfeld** [comment]
I think I've corrected all of these.

**Pull request merged**